proposals/stage-2/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -1,55 +1,49 @@
-# Project Name
+# ${Project Name}
 
 ## Project Champion
 
 Who is the main point of contact during the application process?
 
-## CPC Champion (optional)
-
-If you have a champion from the CPC please identify them. If you do not currently have one we will identify someone to support your project.
-
 ## Description
 
-A rough description of the project
+A rough description of the project in less than 100 words.
 
 ## Statement of alignment with OpenJS Foundation charter and mission
 
-Please refer to [TBD][]
-
-## Current Code of Conduct
+Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
+Keep answers to less than 250 words.
 
-Include a link
+## List of all repos that are part of the project
 
-## Current Project License
-
-Include a link
-
-## Current Source Control
-
-Link to Github or other source control
+Include links and license information
 
 ## Issues Tracker
 
 Link to GitHub or other tracker
 
-## External Dependencies
+## Current Code of Conduct
 
-Full list of dependencies in your source tree including licenses.
+Include a link
 
-## Release Methodology and mechanics
+## Vendored Dependencies (Optional)
 
-How often do you release and where are the releases made. Do you have an LTS program?
+Full list of dependencies in the source tree of all repos. Please include licenses if you can.
+A full audit of all vendored dependencies will be required before your project can officially
+join the foundation, but this can be done later with foundation support.
 
-## List of Committers
+## List of Committers (Optional)
 
-Full list of individuals with a commit bit to your project and their corporate affiliation if applicable.
+Full list of individuals with commit access to your project repos and their corporate affiliation
+if applicable. A full audit of all collaborators will be required before your project can officially
```

Comment:
A full audit of all collaborators sounds scary. What do you mean by this and what's its purpose?

Reply:
I think the general idea was having an idea of the full list of people who had commit access. Perhaps this isn't necessary. 

Was trying to capture the max data we might need and it isn't a big issue to scale back.

Reply:
I've removed this section